File: tests/syn/csharp/01-kitchen.cs
using System;
/// docs
// ordinary line comment
/* TODO block */
[Obsolete]
public sealed record Box<T>(T Value) {
  const ulong Mask = 0xCA_FEUL;
  decimal Rate = 1.25e2M;
  bool Ready = true;
  object Missing = null;
  char Newline = '\n';
  string Basic = "x\\u0041";
  string Verbatim = @"a ""quote""";
  string Interp = $"value={{literal}} {Value ?? default} escape \\n";
  string Both = $@"path {{literal}} {Value} ""ok""";
  public async System.Threading.Tasks.Task<int> Run(int x) { if (x >= 0) return await Next(x); throw new Exception(); }
}
*/

File: tests/syn/java/01-kitchen.java
package demo;
import java.util.List;
/// line docs
// ordinary line comment
/** @param value docs */
/* TODO block */
@Deprecated
public sealed class Box<T extends Number> permits TinyBox {
  private static final long MASK = 0xCA_FEL;
  double ratio = 1.25e-3D;
  boolean ready = true;
  Object absent = null;
  char newline = '\n';
  String escaped = "x\u0041\q";
  int run(int value) { if (value >= 0) return value++; else throw new IllegalArgumentException(); }
}
*/
